### PR TITLE
Adds travis pulp-smash tests

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# https://docs.travis-ci.com/user/trusty-ci-environment/
+dist: trusty
+language: python
+python:
+    # python versions used in el7 SCL & supported fedora
+    - "3.5"
+    - "3.6"
+env:
+  - DB=sqlite TEST=pulp_file
+  - DB=postgres TEST=pulp_file
```
Comment (daviddavis) [inline .travis.yml]: Minor thing but you could probably ditch the `TEST` variables

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+matrix:
+  fast_finish: true
```
Comment (daviddavis) [inline .travis.yml]: This does nothing

File: .travis/before_script.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+if [ "$DB" = 'postgres' ]; then
+  sed -i "s/'ENGINE': 'django.db.backends.sqlite3'/'ENGINE': 'django.db.backends.postgresql_psycopg2'/" pulp/pulpcore/pulpcore/app/settings.py
+  sed -i "s/'USER': ''/'USER': 'pulp'/" pulp/pulpcore/pulpcore/app/settings.py
+  sed -i "s/\/var\/lib\/pulp\/sqlite3.db/pulp/" pulp/pulpcore/pulpcore/app/settings.py
+  psql -U postgres -c 'CREATE USER pulp WITH SUPERUSER LOGIN;'
+  psql -U postgres -c 'CREATE DATABASE pulp OWNER pulp;'
+fi
+
+mkdir -p ~/.config/pulp_smash
+cp .travis/pulp-smash-config.json ~/.config/pulp_smash/settings.json
```
[the remark this answers is not in the document]
Comment (dkliban) [reply .travis/before_script.sh]: make sense ... will do

**Comment by daviddavis** [comment]
@dkliban can you update Jenkins to stop testing PRs?

**Pull request merged**